test/Microsoft.ML.Benchmarks/StochasticDualCoordinateAscentClassifierBench.cs

```diff
@@ -31,31 +29,42 @@ public class StochasticDualCoordinateAscentClassifierBench : WithExtraMetrics
             PetalWidth = 5.1f,
         };
 
-        private Legacy.PredictionModel<IrisData, IrisPrediction> _trainedModel;
-        private IrisData[][] _batches;
```

Comment:
>private IrisData[][] _batches; [](start = 8, length = 30)

`BatchPredictionEngine` is not part of new API, so removing any code related to batch predictions.